README.md

```diff
@@ -1,24 +1,179 @@
-# README
+<a name="readme-top"></a>
 
-This README would normally document whatever steps are necessary to get the
-application up and running.
+## 📗 Table of Contents
 
-Things you may want to cover:
+- [BudgetWise 🍔 ](#BudgetWise--)
+    - [Live Demo ](#live-demo-)
+    - [Tech Stack ](#tech-stack-)
+    - [Key Features ](#key-features-)
+  - [💻 Getting Started ](#-getting-started-)
+    - [Prerequisites ](#prerequisites-)
+    - [Setup ](#setup-)
+    - [Installation ](#installation-)
+    - [database ](#database-)
+    - [Usage ](#usage-)
+    - [Run tests ](#run-tests-)
+  - [✍️ Authors ](#️-authors-)
+    - [👤 **Salwa Ballouti**:](#-salwa-ballouti)
+  - [🔭 Future Features ](#-future-features-)
+  - [🤝 Contributing ](#-contributing-)
+  - [💖 Show your support ](#-show-your-support-)
+  - [🙏 Acknowledgments ](#-acknowledgments-)
+  - [📝  License ](#--license-)
 
-* Ruby version
 
-* System dependencies
+# BudgetWise 🍔 <a name="about-project"></a>
 
-* Configuration
+>**BudgetWise: is a mobile web application where you can manage your budget: you have a list of transactions associated with a category, so that you can see how much money you spent and on what category and when.**.<a name="built-with"></a>
 
-* Database creation
+### Live Demo <a name="live-demo"></a>
+https://budgetwise-4fmb.onrender.com
```

Comment:
- [ ] While I try opening the live demo, It raises the following error as in the image, kindly consider fixing it
>![image](https://github.com/bill7pearl/BudgetWise/assets/53402600/a9ee99fb-2711-4572-a50b-65d46e5067e5)

Reply:
Done, the link is now working.